### ISSUE TITLE
'odin build <file>' does not produce an executable with the correct name

### ISSUE THREAD

**Issue by Tetralux** [odin-lang/Odin#245]
On Windows 10 x64, commit https://github.com/odin-lang/Odin/commit/ec84188597b6558970e66ec7918851f1267888df:

```
> type hello.odin
package hello

using import "core:fmt";

main :: proc() {
    println("Hello, World!");
}

> odin run hello.odin
Hello, World!

> odin build hello.odin

> hello
'hello' is not recognized as an internal or external command,
operable program or batch file.

> main
Hello, World!

**Comment by hazeycode** [comment]
I started looking at this as a way of an introduction to the code base #274

**Comment by hasenj** [comment]
Use the `-out` parameter.

    $ odin build something.odin -out=./something.exe

**Comment by Tetralux** [comment]
I think this was fixed with ae02e9c34aa08b642fd6ca5465f988f8f1d98c60.
